ci/jenkins/jobs/projects.yaml

```diff
@@ -98,6 +98,75 @@
           publishers: []
           included_regions:
           - ci/jenkins/jobs/.*
+      - '{name}-{test_name}-for-pull-request':
+            test_name: integration-pending-label
+            node: null
+            description: 'This is for marking PR as pending for integration test.'
+            builders:
+                - builder-pending-label
+            trigger_phrase: null
+            allow_whitelist_orgs_as_admins: false
+            admin_list: []
+            org_list: []
+            white_list: []
+            only_trigger_phrase: false
+            trigger_permit_all: true
+            status_context: jenkins-integration
+            status_url: --none--
+            success_status: Pending test. Mark as failure. Add comment /test-integration to trigger.
+            failure_status: Pending test. Mark as failure. Add comment /test-integration to trigger.
+            error_status: Pending test. Mark as failure. Add comment /test-integration to trigger.
+            triggered_status: null
+            started_status: null
+            wrappers: []
+            publishers: []
+      - '{name}-{test_name}-for-pull-request':
+            test_name: integration
+            node: 'antrea-test-node'
+            description: 'This is the {test_name} test for {name}.'
+            builders:
+              - builder-integration
+            trigger_phrase: .*/test-(integration|all).*
+            allow_whitelist_orgs_as_admins: true
+            admin_list: '{antrea_admin_list}'
+            org_list: '{antrea_org_list}'
+            white_list: '{antrea_white_list}'
+            only_trigger_phrase: true
+            trigger_permit_all: false
+            status_context: jenkins-integration
+            status_url: null
+            success_status: Build finished.
+            failure_status: Failed. Add comment /test-integration to re-trigger.
+            error_status: Failed. Add comment /test-integration to re-trigger.
+            triggered_status: null
+            started_status: null
+            wrappers:
+                - timeout:
+                      fail: true
+                      timeout: 10
+                      type: absolute
+            publishers: []
+      - '{name}-{test_name}-for-pull-request':
+            test_name: integration-skip
```

Comment:
Maybe we shouldn't have the ability to skip integration tests at all?

Reply:
We can make it be triggered for each pull request since there are not many PRs on CI or docs. But if many PRs come at the same time, checks will queue since we limit the concurrency of these Jenkins jobs. How do you like the idea? @edwardbadboy

Reply:
Yes I think triggering this job automatically is the way to go.

Reply:
Discussed with Zhengsheng, we can make it automatic. However, considering IP allocation, instead of creating a VM each time, using snapshot may be a better choice. Each build will revert the VM back to initial state with snapshot in the beginning.

Reply:
Thanks, the snapshot idea makes sense to me